bin/vernemq.sh

```diff
@@ -339,4 +339,5 @@ trap 'sigterm_handler' SIGTERM
 # Start VerneMQ
 /vernemq/bin/vernemq console -noshell -noinput $@ &
 pid=$!
+/vernemq/bin/join_cluster > /var/log/vernemq/log/join_cluster.log &
```

Comment:
Just asking to be sure. Let's say we only want to start a single node in Docker (no Kubernetes). We'll run the `join_cluster` script so we have to do the right thing for any possible situation.

Reply:
yea, currently the `join_cluster` script will start, see that they are not in an environment that it would require joining for, and shutdown. But I can also make the `join_cluster` script execute conditionally? Thoughts?

Reply:
@ioolkos I went ahead and updated it to conditionally start based on it needing to join a cluster.